src/main/java/com/marklogic/mgmt/resource/clusters/ClusterManager.java

```diff
@@ -85,7 +87,8 @@ public void addHost(AdminManager adminManager, String hostname) throws Exception
     public void addHost(AdminManager adminManager, String hostname, String group, String zone) throws Exception{
     	AdminConfig adminConfig = adminManager.getAdminConfig();
 
-		AdminConfig joiningHostAdminConfig = new AdminConfig(hostname, 8001, adminConfig.getUsername(), adminConfig.getPassword());
+		AdminConfig joiningHostAdminConfig = new AdminConfig(adminConfig);
```

Comment:
Switched over to the copy constructor, which uses all possible connection/auth properties, as opposed to assuming usage of username/password.